.travis.yml

```diff
@@ -2,13 +2,14 @@
 language: go
 
 go:
-  - 1.4
+  - 1.5
```

Comment:
maybe a comment about how 1.4 is **probably** supported, we just don't test it because glied?